modules/syslog.lua

```diff
@@ -13,10 +13,16 @@ local tonumber = tonumber
 local error = error
 local type = type
 local ipairs = ipairs
+local rawset = rawset
```

Comment:
Sorry, I forgot a commit in this PR. `rawset` is used, and the `l.C` seems necessary.